runtime/ftplugin/query.lua

```diff
@@ -0,0 +1,5 @@
+-- Neovim filetype plugin file
+-- Language:	Tree-sitter query (Scheme)
+-- Last Change:	2022 Mar 29
+
+vim.bo.commentstring = "; %s"
```

Comment:
I would omit Scheme. Apart from the file ending and the parens they have nothing in common.

Reply:
Why is the file ending `.scm` then?

Reply:
I have no idea about ftplugins but one would probably also set to lisp indentation

Reply:
Should I just `!runtime ftplugin/lisp.vim`? (And similar for `indent/query.lua`)?

Reply:
Because they needed a file ending and they used a scheme-like language. The docs also jokingly say that it maybe read as "source code matching" ;-) https://tree-sitter.github.io/tree-sitter/syntax-highlighting#queries